### ISSUE TITLE
Version 1.0.5 does not match Github Repo

### ISSUE THREAD

**Issue by tomjschwanke** [corona-warn-app/cwa-app-android#847]
## Avoid duplicates
* [X] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Question is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Question has not already been asked in another issue

## Your Question

Yesterday I downloaded the update to 1.0.5 through the playstore, but it seems to be the same as 1.0.4. #723 lists a lot of fixes under "1.0.5", including one I proposed, so I assumed they'd be in the update but they weren't.
I still can't click the row to toggle and it also still shows only 13 of 14 days active. 

Was that on purpose?

**Comment by pwoessner** [comment]
Hi @tomjschwanke 
you are right. The PR that was merged into master with the label 1.0.5 contains features and fixes that we want to deliver as a current result of our sprint. Due to an unforeseen issue we had to deliver a hotfix: https://github.com/corona-warn-app/cwa-app-android/releases/tag/1.0.5 . This hotfix only fixes the high prio bug and does not include any other improvements. Therefore we had to increase the version number to 1.0.5 and we are working now internally on 1.0.6 which includes also the fixes that you provided. 
We are currently also working on our repo setup to make hotfixes more clearer to you and have only the current prod coding that will be submitted to the play store on our master branch. 

Sorry for the confusion, I have edited the PR title to 1.0.5-SNAPSHOT, which makes it perhaps a little bit clearer, that this was an internal test release that was merged into master.

Thanks,
Philipp
